include/boost/geometry/index/detail/varray.hpp

```diff
@@ -661,14 +644,14 @@ class varray
     //!
     //! @par Complexity
     //!   Constant O(1).
-    void push_back(BOOST_RV_REF(value_type) value)
+    void push_back(value_type && value)
```

Comment:
I think you should avoid space before `&&` to be consistent. Also in other places in this PR.